app/boards/shields/m60/Kconfig.defconfig

```diff
@@ -0,0 +1,10 @@
+
+if SHIELD_M60
+
+config ZMK_KEYBOARD_NAME
+	default "m60"
+
+config ZMK_KSCAN_MATRIX_POLLING
+	default y
```

Comment:
Is this in fact needed? Does interrupt based not work? If not, that would be a bummer, since it'll be much less power efficient.

If so, this should hopefully be temporary, the work I'm doing to enable using PORT events should remove the 8-max IN interrupt problem I *suspect* you might have hit, if you have this here.

Reply:
Oops, this is an oversight. I was trying ZMK with black pill f411 boards and used this config before i receive the M60.
When porting the code, copied all without thinking much.
Will try once whether all keys working without this polling stuff and remove if not needed.

Reply:
Yes its not needed. Removed it